### PR TITLE
Clean up example showing how to mix force fields and export to AMBER/GROMACS files

### PR DESCRIPTION
**Pull request by andrrizzi**
The new example is in `examples/conversion_amber_gromacs/convert_to_amber_gromacs.ipynb`.

### CODE REVIEW

**Comment by j-wags** [comment]
This pull request **fixes 1 alert** when merging 9111ebf7e6215e1496566efe0bc4473f8aae20bd into 2e86a5c9a37a0334137754911b21b6864ec97152 - [view on LGTM.com](https://lgtm.com/projects/g/openforcefield/openforcefield/rev/pr-4264ae60c11a29466ecea817d92743e90b173fb6)

**fixed alerts:**

* 1 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: examples/conversion_amber_gromacs/1_cyclohexane_1_ethanol.pdb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+REMARK   1 CREATED WITH MDTraj 1.7.2, 2016-06-27
```
Comment (j-wags) [inline examples/conversion_amber_gromacs/1_cyclohexane_1_ethanol.pdb]: As a note to other readers: I was initially on the fence about adding copies of the molecule files to the example folders, but I think users will find it much clearer to have the molecules in here where they can investigate them.

File: openforcefield/tests/utils.py [hunk 2]
```diff
@@ -471,11 +471,19 @@ def compare_system_energies(system1, system2, positions, box_vectors=None,
 
     # Group force types into OpenMM force groups.
     if by_force_type:
+        # Remove eventual CMMotionRemover forces that do not affect energies.
```
Comment (j-wags) [inline openforcefield/tests/utils.py]: Good catch.

**Comment by andrrizzi** [comment]
Fantastic, thank you for the review!

**Pull request merged**